### PR TITLE
feat(blockchain): Add support for new Cluster Governance blockchain API

### PR DESCRIPTION
**Pull request by skambalin**
- Update blockchain module to support changes added in `v5.4.0`
- Add new `DDCClusterGovPallet` pallet class
- Fixed the SDK tests bootstrap and blockchain tests
- Updated blockchain module docs

### CODE REVIEW

File: packages/blockchain/src/DDCClusterGovPallet.ts [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+ * const clusterId = '0x...';
+ * const protocolParams = { ... };
+ * const member = ClusterMember.ClusterManager;
+ *
+ * const tx = blockchain.ddcClusterGov.proposeUpdateClusterProtocol(clusterId, protocolParams, member);
+ *
+ * await blockchain.send(tx, { account });
+ * ```
+ */
+export class DDCClusterGovPallet {
```
[the remark this answers is not in the document]
Comment (skambalin) [reply packages/blockchain/src/DDCClusterGovPallet.ts:21]: Will update. Thanks.

**Review by upalinski — APPROVED** [review]

File: packages/blockchain/src/DDCClusterGovPallet.ts [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+  /**
+   * Creates a local proposal within a cluster, intended to update protocol parameters (like validators, treasury fee, pricing for stored and streamed bytes, etc.) of a previously activated cluster in the network.
+   * Only cluster members, such as the cluster manager and node providers, can vote on this type of proposal.
+   * Any cluster member has the ability to create this type of proposal.
+   *
+   * @param clusterId - The ID of the cluster.
+   * @param protocolParams - The new protocol parameters.
+   * @param member - The member who is creating the proposal.
+   * @param podePublicKey - The public key of the node provider. Needed in case the member is ClusterMember.NodeProvider.
```
Comment (yahortsaryk) [inline packages/blockchain/src/DDCClusterGovPallet.ts:43]: typo in `podePublicKey` -> `nodePublicKey`

File: packages/blockchain/docs/classes/DDCClusterGovPallet.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+const tx = blockchain.ddcClusterGov.proposeUpdateClusterProtocol(clusterId, protocolParams, member);
+
+await blockchain.send(tx, { account });
+```
+
+## Methods
+
+### closeProposal
+
+▸ **closeProposal**(`clusterId`, `member`, `podePublicKey?`): `Sendable`
```
Comment (yahortsaryk) [inline packages/blockchain/docs/classes/DDCClusterGovPallet.md:23]: typo `podePublicKey` -> `nodePublicKey`

File: packages/blockchain/docs/classes/DDCClusterGovPallet.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+const tx = blockchain.ddcClusterGov.retractProposal(clusterId);
+
+await blockchain.send(tx, { account });
+```
+
+___
+
+### voteProposal
+
+▸ **voteProposal**(`clusterId`, `approve`, `member`, `podePublicKey?`): `Sendable`
```
Comment (yahortsaryk) [inline packages/blockchain/docs/classes/DDCClusterGovPallet.md:175]: typo `podePublicKey` -> `nodePublicKey`

File: packages/blockchain/src/DDCClusterGovPallet.ts [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+   * ```typescript
+   * const clusterId = '0x...';
+   * const member = ClusterMember.ClusterManager;
+   *
+   * const tx = blockchain.ddcClusterGov.retractProposal(clusterId, member);
+   *
+   * await blockchain.send(tx, { account });
+   * ```
+   */
+  closeProposal(clusterId: ClusterId, member: ClusterMember, podePublicKey?: NodePublicKey) {
```
Comment (yahortsaryk) [inline packages/blockchain/src/DDCClusterGovPallet.ts:121]: typo in `podePublicKey` -> `nodePublicKey`

File: packages/blockchain/src/DDCClusterGovPallet.ts [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+    return this.apiPromise.tx.ddcClusterGov.closeProposal(
+      clusterId,
+      this.createClusterMember(member, podePublicKey),
```
Comment (yahortsaryk) [inline packages/blockchain/src/DDCClusterGovPallet.ts:124]: typo in `podePublicKey` -> `nodePublicKey`

**Review by yahortsaryk — APPROVED** [review]

**Pull request merged**